### ISSUE TITLE
Not seeing files in user home directory

### ISSUE THREAD

**Issue by akblissweb** [stilliard/docker-pure-ftpd#194]
Sure this maybe a newbie mistake somewhere but I've setup a container with docker compose. Set my directories.  I can go inside the container and see html files inside the home directory. 

I can connect from the ftp client, it logs in, lists the directory and finds nothing there.

I didn't run any scripts, I don't have a Dockerfile. I just set the user using the environment variables in the yml file.

What could I be missing ?  Any debugging steps I can take ?

**Comment by stilliard** [comment]
Hi @akblissweb 

Interesting, I'd first double check the directories are correct. 

Would you mind sharing your docker compose setup please?

**Comment by akblissweb** [comment]
Yay, it seems I have solved it just by thinking about your comment and others.
I decided to try:

Change the Username
Change the Home Directory 
Delete the passwd file 
Delete the container
Restart everything

Then the files appeared ok.

I guess I must have created the yml file with one user/directory and started it up and then maybe changed the user/directory in the yml file after that.  Broke it nicely.
